release/models/network-instance/openconfig-network-instance.yang

```diff
@@ -633,7 +641,11 @@ module openconfig-network-instance {
               address families enabled, the protocol=BGP,
               address-family=IPv4 table is created by the system. The
 	            removal of the table should not require additional or
-	            explicit configurations";
+	            explicit configurations.
+
+              Users are not expected to create or delete tables.
```

Comment:
nit: Perhaps pedantic but since we’re clarifying; to me saying users are not expected to do a thing implies that they still able to do that thing but it is not required.

Reply:
That's fair.